SparkleSpin/Views/EntryCell/EntryCell.swift

```diff
@@ -6,10 +6,44 @@
 //  Copyright © 2019 Britney Smith. All rights reserved.
 //
 
+enum CellState {
+    case selected
+    case saved
+}
+
 import UIKit
 
 class EntryCell: UITableViewCell {
 
-   
+    @IBOutlet weak var entryCellLabel: CustomCellLabel!
+    
+    
+    var item: PlayerModel? {
+        didSet {
+            entryCellLabel.text = item?.name
+        }
+    }
+    
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+        
+        if selected {
+            setCellStateWith(state: .selected)
+        } else {
+            setCellStateWith(state: .saved)
+        }
```

Comment:
If we are using only the selected state of the cell to determine the background the I would say we can save some lines of code by removing that enum and moving the logic of your `setCellStateWith ` function here. What do you think?

Reply:
It looks like I'm also using `setCellStateWith()` to set the state of the a newly inserted row in `insertNewPlayerRowInTable()` (in `PlayerViewController`). It may still be convenient to keep the cell selected state view logic in another method.

Also, I'm thinking about taking out the selected state altogether in another PR, since the selected view changes on the cell aren't preserved by the Cocoa framework by default when swiping to delete. Doesn't seem like it has a purpose here knowing that. So, this might be moot anyway!

Reply:
Decided to remove the custom selected cell state based on what I said above. Changes in https://github.com/BritneyS/SparkleSpin/pull/10/commits/5c4ec413668cf697262d172c3675577a8ec8bda8